### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by sebibhutta** [codesandbox/codesandbox-client#6374] [Q&A]
I have taken pull of the code.
on local it is running very smoothly.
but when i create build of this project and upload to xampp it does not access component routes, like is is giving error on "/s" , "/signin". it thinks that they are folders but they are components. but if there is folder like pricing it is accessing it.
i am facing this and trying to run this project on my xampp but nothing works.
anyone please help me to run through xampp.

**Comment by JamesACS** [comment]
Hey @sebibhutta - I'm sorry to hear about the trouble. 

Please could you give us some more information about the trouble and maybe a link to the Sandbox? What are you trying to do? Are there any more specific error messages? 

Alternatively, you may find our Discord helpful for coding help! 

https://discord.gg/A4MjgPan

**Reply by sebibhutta** [reply]
I am explaining again.
I have taken pull from https://github.com/codesandbox/codesandbox-client.git on my machine.
first i have install all the packages by "yarn" command.
then i have executed command "npm start" and then it shows that your project is running on "localhost:3000" this is fine.

now i want to create a build of this project by "yarn build"
and it creates almost two or three builds.
i have uploaded www from root to my apache server (xampp).
and when i click on " /pricing " it opens the page for me... but the issue is when i click on " /signin  and Create Sandbox Button" it shows 404.

i think there is some issue with the build. please help

**Reply by sebibhutta** [reply]
How to place builds hierarchy in my apache server (xampp) ??

**Reply by sannek** [reply]
The routes that are not working, are ones that normally get redirected to the server, but the configuration for that is not open source. So those routes just get treated like regular client site routes in your build, and therefore won't work as expected if you deploy your own version of CSB client.

**Reply by sebibhutta** [reply]
Okay got it. thanks.
